src/foam/java/refinements.js

```diff
@@ -672,6 +687,13 @@ foam.CLASS({
       var info = this.SUPER(cls);
       var compare = info.getMethod('compare');
       compare.body = this.compareTemplate();
+
+      info.method({
+        name: 'getOf',
```

Comment:
How/where is this used? Is it needed?

If 'of' is set, shouldn't it change the 'javaType' above?